### DISCUSSION TITLE
Use targeted instead of selected for damage

### DISCUSSION THREAD

**Discussion by gregpola** [fvtt-fria-ligan/twilight2000-foundry-vtt#110] [Ideas]
Any chance we could get the apply damage logic changed to apply it to the targeted token instead of the selected token? This is one of the least intuitive things about Foundry, makes no sense to not apply it to the target.

**Comment by tinwe** [comment]
Agree. Our players often applies it to the wrong character because of this.
Although, one should probably keep the selected option as a backup when no token is targetted, to please both kinds of playstyle.

**Comment by tinwe** [comment]
My players aren't able to apply damage themselves. It says they lack permissions.
Is this by design, so that only the Referee can apply the damage, or what settings should I enable for them to be able to apply the damage themselves?
I really don't want to give all players ownership to the NPC characters nor their tokens, for this.

**Comment by Stefouch** [comment]
It is a design issue, because Foundry "sees" that a player is trying to apply a change (damage) to an NPC (controlled by the GM) and since that player does not have permission over the NPC (obviously), Foundry forbids the change.
